apps/web/src/utils/ioc.ts

```diff
@@ -3,17 +3,23 @@ import { modules } from '@walless/ioc';
 import { createEncryptionKeyVault } from '@walless/messaging';
 import { configure, create } from '@walless/store';
 import IDBPouch from 'pouchdb-adapter-idb';
+import { initializeAuth } from 'utils/firebase';
+
+import { makeConfig } from '../../scripts/kernel/utils/config';
```

Comment:
Is `makeConfig` need to be put into common utils?

Reply:
yes, because it is used in more than 1 single place